py_proxy/app.py

```diff
@@ -5,40 +5,38 @@
 
 from py_proxy.sentry_filters import SENTRY_FILTERS
 
+REQUIRED_PARAMS = ["client_embed_url", "nginx_server", "legacy_via_url"]
 
-def settings():
-    """Return the app's configuration settings as a dict.
 
-    Settings are read from environment variables and fall back to hardcoded
-    defaults if those variables aren't defined.
+def load_settings(settings):
```

Comment:
Probably a nit-you can take it or leave it. When I see load_settings it makes me think they are being loaded into pyramid but really they are just being grabbed from the env/added to the dict of settings here. I think `get_settings` or `generate_settings` might make more sense in this scenario but I'll leave it up to you.

Reply:
I think "get_settings" is better, I'll change it. I just wanted to differentiate between *doing* something about settings and *being* the settings.